jsonrpc/src/main.ts

```diff
@@ -269,7 +269,7 @@ export interface LogTraceParams {
 }
 
 export namespace LogTraceNotification {
-	export const type = new NotificationType<LogTraceParams, void>('$/logTraceNotification');
+	export const type = new NotificationType<LogTraceParams, void>('logTrace');
```

Comment:
Actually we should name this `$/logTrace` since clients and servers don't need to implement this.